e107_web/js/core/front.jquery.js

```diff
@@ -80,12 +80,15 @@ $(document).ready(function()
 
     $(document).on("click", ".e-comment-submit", function(){
 			
+			var commentsParent = $(this).parent().parent().parent().parent().parent().parent().parent();

```

Comment:
This won't work if you use custom templates for comments, because the HTML structure may be not the same. Use parents() instead. For example if you want to get the form, which contains your element, use something like this:

```javascript
var $form = $(this).parents('form:first');
```